### ISSUE TITLE
Math operations with comments in between fail

### ISSUE THREAD

**Issue by sepptruetsch** [sass/node-sass#1242]
Using 
```css
min-width: 20px /* height */ + 2*5px /* padding */ + 2*1px /*border*/;
```
results in 
```css
min-width: 20px 10px 2px;
```

The comments in between mess it up as 
```css
min-width: 20px + 2*5px + 2*1px; /* height + padding + border */
```
works fine.

Thanks for your time and effort!

**Comment by xzyfer** [comment]
Thanks for the report @sepptruetsch. I can confirm this is a bug with LibSass. I've created an issue to track it with them - https://github.com/sass/libsass/issues/1765

**Comment by xzyfer** [comment]
This is fixed in the upcoming 3.5.0-beta. Subscribe to #1349 for further updates.
